staff/lluis-bagur/CV/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
folder should not be in upper-case, but lower-case, always, please, pay attention and take action

Reply:
Solved!